kuma/wiki/tests/conftest.py

```diff
@@ -7,6 +7,11 @@
 from ..models import Document, Revision
 
 
+class Object(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
I prefer [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) for packing, but I think others (@jgmize?) prefer this, so I can get used to it.